app/views/unattended/poap/disklayout.erb

```diff
@@ -0,0 +1,5 @@
+<%#
+kind: ptable
+name: NX-OS default fake
+%>
+# Not required for NX-OS POAP provisioning.
```

Comment:
If the template is not required, just remove it? Default templates of any kind are normally added to [community-templates](https://github.com/theforeman/community-templates) then pulled here.